samplestore/src/main/java/com/stripe/samplestore/service/StripeService.java

```diff
@@ -0,0 +1,24 @@
+package com.stripe.samplestore.service;
+
+import java.util.Map;
+
+import okhttp3.ResponseBody;
+import retrofit2.http.Body;
+import retrofit2.http.FieldMap;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.POST;
+import rx.Observable;
+
+/**
+ * The {@link retrofit2.Retrofit} interface that creates our API service.
+ */
+public interface StripeService {
+
+    @POST("charge")
```

Comment:
QQ: does it matter whether or not the create charge post item is `@FormUrlencoded`?

Reply:
what do you mean by matter?
we're sending in via the body now and not via form url encoded so that's gone. theoretically our backend still takes the params via form url encoded (but we were running into the strings issue). you can pass a map via form url encoded using the @fieldmap annotation.

hahaha did something in there answer the intended the question?

Reply:
If it's just the body, then it probably doesn't matter -- I was trying to think of whether or not we needed to have hat annotation anyhow, but i think that would only be the case if we were also adding url parameters.